src/Compilers/CSharp/Portable/Symbols/NamedTypeSymbol.cs

```diff
@@ -1261,23 +1261,25 @@ internal virtual FieldSymbol FixedElementField
         internal abstract bool IsInterface { get; }
 
         /// <summary>
-        /// Verify if the given type can be used to back a tuple type and return cardinality of that tuple type in <paramref name="tupleCardinality"/>. 
+        /// Verify if the given type can be used to back a tuple type 
```

Comment:
Nevermind, I see the change right below (returning false if type is already a tuple).